comments-bundle/contao/templates/comments/com_default.html5

```diff
@@ -8,7 +8,7 @@
     <div class="reply">
       <p class="info"><?= $this->rby ?> <?= $this->author->name ?></p>
       <div class="comment">
-        <?= $this->reply ?>
+        <?= $this->extractStyleAttributesForCsp($this->reply) ?>
```

Comment:
Can we maybe find a more intuitive name? Of course extracting the style attributes for CSP is what the filter does, but the users would maybe find something like `$this->richText($this->reply)` more intuitive.

Reply:
```suggestion
        <?= $this->cspInlineStyles($this->reply) ?>
```

Reply:
I think we should also rename https://github.com/contao/contao/blob/b071911d6680faf836b4fc38175fa3b89e294af8/core-bundle/contao/library/Contao/TemplateTrait.php#L197 to `cspInlineStyle()` /cc @fritzmg

Reply:
I suggested not to call it `cspInlineStyle()`. 🙈 But I have no problem with that, either, if you prefer.

Reply:
Yes, I strongly prefer it to have `csp` in it’s name for security reaseons.

Reply:
Yeah, it was `cspInlineStyle` originally. But I am also fine with renaming it back to `cspInlineStyle` (+ `cspInlineStyles` for the new method).